docs/_chapters/330-jpms.md

```diff
@@ -127,3 +129,53 @@ public class Main {
 ### Java 8 Support
 
 Bnd's `module-info.class` generation is supported when building with Java 8 or higher. (Yes! I did say Java 8.)
+
+## Advanced Options
+
+There are scenarios where the heuristics used by bnd don't give the desired result because the necessary information is not available or is incorrect.
+
+The `-jpms-module-info-options` instruction provides some capabilities to help the developer handle these scenarios. The instruction uses the _package header syntax_ similar to many other bnd instructions. The keys of these instructions are module names and there are 4 available attributes. They are:
+
+- **`from`** - A module name generated by bnd that should be replaced with the key of the instruction.
```

Comment:
Maybe `substitute` would be a better name. You should make constants for these strings too.

Reply:
k